contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -187,7 +175,9 @@ def filter_assessments(self, queryset, name, value):
         ).exclude(assessment_count=0)
 
     def filter_subtitles(self, queryset, name, value):
-        subtitle_query = self.main_tree_query.filter(files__preset__subtitle=True)
+        subtitle_query = self.main_tree_query.filter(
+            files__preset__subtitle=True, kind="video"
```

Comment:
For consistency, let's use `content_kinds.VIDEO` here.

Reply:
@rtibbles thanks, done!